src/api/tags/tag.ts

```diff
@@ -40,5 +40,6 @@ export const enum TagPathsType {
   aws = 'aws',
   azure = 'azure',
   gcp = 'gcp',
+  ibm = 'ibm',
```

Comment:
I think this will have to be "gcp" for now. Don't believe the tags API currently supports IBM?

Reply:
done